contracts/adapters/SFCToGovernable.sol

```diff
@@ -31,11 +31,22 @@ interface SFC {
 
     /// @dev Get the sum of all active delegated stakes.
     function getTotalActiveStake() external view returns (uint256);
+
+    /// @notice Update slashing refund ratio for a validator.
+    /// @dev The refund ratio is used to calculate the amount of stake that can be withdrawn after slashing.
+    function updateSlashingRefundRatio(uint256 validatorID, uint256 refundRatio) external;
+
+    /// @notice Check whether the given validator is slashed
+    function isSlashed(uint256 validatorID) external view returns (bool);
 }
 
 // @dev SFCToGovernable is an adapter allowing to use the network SFC contract as Governable (governance votes weights provider).
 contract SFCToGovernable is Governable {
```

Comment:
The conventional name should be `SfcGovernableAdapter`, maybe `SFCGovernableAdapter`